rules/jsx-import.js

```diff
@@ -0,0 +1,126 @@
+'use strict';
+const arrayToSentence = require('array-to-sentence');
+const astUtils = require('eslint-ast-utils');
+const getDocsUrl = require('./utils/get-docs-url');
+
+const pragmas = {
+	h: ['dom-chef', 'preact'],
+	React: ['react'],
+	preact: ['preact']
+};
+
+const pragmaAPIs = {
+	React: ['cloneElement']
+};
+
+const getSuggestedPackages = modules => arrayToSentence(modules.map(x => `\`${x}\``), {lastSeparator: ' or '});
+
+const hasPackage = (node, packages) => node.init ?
+	packages.some(x => x === astUtils.getRequireSource(node.init)) :
+	packages.some(x => x === node.parent.source.value);
+
+const isImportDeclaration = node => node.init ?
```

Comment:
This could be moved to a `index.js` file in https://github.com/sindresorhus/eslint-plugin-unicorn/tree/master/rules/utils